src/library/scanner/libraryscanner.h

```diff
@@ -55,13 +55,11 @@ class LibraryScanner : public QThread {
                    ConfigObject<ConfigValue>* pConfig);
     virtual ~LibraryScanner();
 
+  public slots:
     // Call from any thread to start a scan. Does nothing if a scan is already
     // in progress.
     void scan();
```

Comment:
GitHub has tricked me: 

> 1,265 src/mixxx.cpp
>153 additions, 1,112 deletions not shown 

Yes here it is: 
https://github.com/rryan/mixxx/blob/menubar/src/mixxx.cpp#L419